.github/workflows/release.yaml

```diff
@@ -6,14 +6,29 @@ permissions: write-all
 env:
```

Comment:
afaict, your workflow requires this permissions now:

contents: write
packages: write

you could change from write-all to that list.

Reply:
Thanks for pointing out this. Has restricted to a smaller privilege.